libs/utils/decorate.js

```diff
@@ -308,3 +306,22 @@ export function decorateMultiViewport(el) {
   }
   return foreground;
 }
+
+export function turnAnchorIntoVideo({ src, anchorTag }) {
+  if (anchorTag.closest('.marquee, .aside, .hero-marquee') && !anchorTag.hash) anchorTag.hash = '#autoplay';
+  const { dataset, parentElement } = anchorTag;
+  const attrs = getVideoAttrs(anchorTag.hash, dataset);
+  const video = `<video ${attrs} data-video-source=${src}></video>`;
```

Comment:
Not sure we need the extra `attrs` const here, we can directly do
```suggestion
  const video = `<video ${getVideoAttrs(anchorTag.hash, dataset)} data-video-source=${src}></video>`;
```